.changeset/hip-paws-run.md

```diff
@@ -0,0 +1,6 @@
+---
+"@salt-ds/core": minor
+"@salt-ds/lab": minor
+---
+
+Promote splitter to core
```

Comment:
A Core user would see this and ask what is Splitter ?
Have a look through the changelog and see how other components have been announced in Core's changelog.
Something like

`Add `Dialog`, `DialogHeader`, `DialogContent`, `DialogActions`, and `DialogCloseButton` to core`

or Dropdown's changelog entry

we then give an example of the code